proposals/lip-0052.md

```diff
@@ -673,9 +669,10 @@ def execute(
             NFTStore(nftID).attributesArray = getNewAttributes(nftID, storedAttributes, receivedAttributes)
             createUserEntry(recipientAddress, nftID)
             delete entry escrowEntry(sendingChainID, nftID) from the escrow substore
-        else: # message is returning to the non-native chain
-            createNFTEntry(senderAddress, nftID, receivedAttributes)
+        else: # message is returning to the native chain
```

Comment:
I was thinking that this handling is correct only if the ccm status is one set byy Interoperability (so that it is guaranteed that sending and receiving chains have been swapped). I think we should explicitly only handle these cases, so I would change `else` to `elif ccm.status <= MAX_RESERVED_ERROR_STATUS` (MAX_RESERVED_ERROR_STATUS is defined in LIP0045).

Reply:
Changed as suggested.